common/static/common/js/spec/components/paginated_view_spec.js

```diff
@@ -0,0 +1,182 @@
+            expectHeader('Showing 1-5 out of 6 total');
+            expectCards(initialItems);
+            expectFooter({currentPage: 1, totalPages: 2, isHidden: false});
+        });
+
+        it('can render the only page', function () {
+            initialItems = generateItems(1, 1);
+            testCollection.set(
+                {
+                    "count": 1,
+                    "num_pages": 1,
+                    "current_page": 1,
+                    "start": 0,
+                    "results": initialItems
+                },
+                {parse: true}
+            );
+            expectHeader('Showing 1 out of 1 total');
+            expectCards(initialItems);
+            expectFooter({currentPage: 1, totalPages: 1, isHidden: true});
+        });
+
+        it('can change to the next page', function () {
+            var requests = AjaxHelpers.requests(this),
+                newItems = generateItems(1, 1);
+            expectHeader('Showing 1-5 out of 6 total');
+            expectCards(initialItems);
+            expectFooter({currentPage: 1, totalPages: 2, isHidden: false});
+            expect(requests.length).toBe(0);
+            testView.$(nextPageButtonCss).click();
+            expect(requests.length).toBe(1);
+            AjaxHelpers.respondWithJson(requests, {
+                "count": 6,
+                "num_pages": 2,
+                "current_page": 2,
+                "start": 5,
+                "results": newItems
+            });
+            expectHeader('Showing 6-6 out of 6 total');
+            expectCards(newItems);
+            expectFooter({currentPage: 2, totalPages: 2, isHidden: false});
+        });
+
+        it('can change to the previous page', function () {
+            var requests = AjaxHelpers.requests(this),
+                previousPageItems;
+            initialItems = generateItems(1, 1);
+            testCollection.set(
+                {
+                    "count": 6,
+                    "num_pages": 2,
+                    "current_page": 2,
+                    "start": 5,
+                    "results": initialItems
+                },
+                {parse: true}
+            );
+            expectHeader('Showing 6-6 out of 6 total');
+            expectCards(initialItems);
+            expectFooter({currentPage: 2, totalPages: 2, isHidden: false});
+            testView.$(previousPageButtonCss).click();
+            previousPageItems = generateItems(1, 5);
+            AjaxHelpers.respondWithJson(requests, {
+                "count": 6,
+                "num_pages": 2,
+                "current_page": 1,
+                "start": 0,
+                "results": previousPageItems
+            });
+            expectHeader('Showing 1-5 out of 6 total');
+            expectCards(previousPageItems);
+            expectFooter({currentPage: 1, totalPages: 2, isHidden: false});
+        });
+
+        it('sets focus for screen readers', function () {
```

Comment:
Is this a relevant test for ``PaginatedView``? Is it relying on the header to do this?